thunder.post_update.php

```diff
@@ -22,3 +24,70 @@ function thunder_post_update_ensure_config_selector_installed() {
   // Output logged messages to related channel of update execution.
   return $thunderUpdater->logger()->output();
 }
+
+/**
+ * Installs the content_content_lock view.
+ */
+function thunder_post_update_add_content_lock_view() {
+
+  $content_view = View::load('content');
+  if (empty($content_view)) {
+    return;
+  }
+  $content_view->setThirdPartySetting('config_selector', 'feature', 'thunder_content_view');
+  $content_view->setThirdPartySetting('config_selector', 'priority', 0);
+
+  $content_view->calculateDependencies();
+
+  if (in_array('content_lock', $content_view->getDependencies()['module'])) {
+    // Duplicate content into content_content_lock view and add third party
+    // settings.
+    $content_lock_view = $content_view->createDuplicate();
+    $content_lock_view->set('id', 'content_content_lock');
+    $content_lock_view->setThirdPartySetting('config_selector', 'feature', 'thunder_content_view');
+    $content_lock_view->setThirdPartySetting('config_selector', 'priority', 1);
+    $content_lock_view->save();
+
+    // Delete all content_lock related stuff from the content view.
```

Comment:
I don't get why we do this. Their current view is now the content lock view - why don't we just import the non-content lock view from the install config and be done.

Reply:
We do that to keep the changes a user has done to the view.